reference-data-proxy/src/v1/routes/healthcheck.route.ts

```diff
@@ -0,0 +1,31 @@
+import express, { Request, Response } from 'express';
+import dotenv from 'dotenv';
+dotenv.config();
+
+export const healthcheck = express.Router();
+const GITHUB_SHA = process.env.GITHUB_SHA || 'undefined';
+
+export const pingMulesoft = async () => 'Not configured';
+
+healthcheck.get('/healthcheck', (req: Request, res: Response) => {
+  const mulesoft = pingMulesoft();
+
+  const data = {
+    uptime: process.uptime(),
+    message: 'Ok',
```

Comment:
make and use constants instead for strings?

Reply:
It may not be necessary here because it's only being used once